xbmc/windowing/android/WinSystemAndroid.cpp

```diff
@@ -144,7 +129,6 @@ bool CWinSystemAndroid::CreateNewWindow(const std::string& name,
 
   m_android->SetNativeResolution(res);
 
-  m_bWindowCreated = true;
```

Comment:
`m_bWindowCreated = true` itself is fine (not need remove this line)

The regression is only because flag is used to exit early when "No need to create a new window" supposedly

Reply:
OK, keep `m_bWindowCreated = true`. I had removed it to completely reverse the change. Let's see if there are any other side effects